src/cli/node/json_rpc_client.rs

```diff
@@ -79,6 +81,79 @@ pub fn get_blockchain(addr: SocketAddr, epoch: i64, limit: i64) -> Result<(), fa
     Ok(())
 }
 
+#[allow(clippy::cast_possible_wrap)]
+pub fn get_supply_info(addr: SocketAddr) -> Result<(), failure::Error> {
+    let mut stream = start_client(addr)?;
+
+    let request = r#"{"jsonrpc": "2.0","method": "getSupplyInfo", "id": "1"}"#;
+    let response = send_request(&mut stream, request)?;
+    let supply_info = parse_response::<SupplyInfo>(&response)?;
+
+    log::info!("{:?}", supply_info);
+
+    println!(
+        "\nSupply info at {} (epoch {}):\n",
+        pretty_print(supply_info.current_time as i64, 0),
+        supply_info.epoch
+    );
+
+    let block_rewards_wit =
+        Wit::wits_and_nanowits(Wit::from_nanowits(supply_info.blocks_minted_reward)).0;
```

Comment:
Create a local function to make this more clear:

```rust
fn whole_wits(nanowits: u64) -> u64 {
    Wit::wits_and_nanowits(Wit::from_nanowits(nanowits)).0
}
```

```suggestion
        whole_wits(supply_info.blocks_minted_reward);
```